Update VERSIONS_YAML_FILE e2e test information (#916)
istio-ecosystem/sail-operator: tests/e2e/README.md

*Important*: avoid adding in the custom file versions that are not available in the `pkg/istioversion/versions.yaml` file. The test framework will not be able to run the tests because the operator does not contains the charts for those versions.
* To run the test framework against a specific Istio version, you can use the following command:
```
$ VERSIONS_YAML_FILE=custom_versions.yaml SKIP_BUILD=true SKIP_DEPLOY=true GINKGO_FLAGS="-v --label-filter=smoke" make test.e2e.kind
```

> Note: the `custom_versions.yaml` file should be located in the `pkg/istioversion` directory. The test framework will use this file to run the tests against the specific Istio versions defined in it.

Reviewer: ```suggestion
Note: The `custom_versions.yaml` file must be placed in the `pkg/istioversion` directory. The test framework uses this file to run tests against the specific Istio versions it defines.
```
